### ISSUE TITLE
_zoomCalc function missing 'this'

### ISSUE THREAD

**Issue by ZuchaoWang** [teralytics/Leaflet.D3SvgOverlay#2]
It seems two 'this.' is missing in the _zoomCalc function. The original code is:

```
    _zoomCalc: function (evt) {
        var newZoom = evt.zoom || map._zoom;
        this._zoomDiff = newZoom - this._zoom;
        this._scale = Math.pow(2, this._zoomDiff);
        this._shift = this.map._latLngToNewLayerPoint(this._origin, newZoom, (evt.center || map._initialCenter ));
    },
```

I think it should be:

```
    _zoomCalc: function (evt) {
        var newZoom = evt.zoom || this.map._zoom; // map._zoom -> this.map._zoom
        this._zoomDiff = newZoom - this._zoom;
        this._scale = Math.pow(2, this._zoomDiff);
        this._shift = this.map._latLngToNewLayerPoint(this._origin, newZoom, (evt.center || this.map._initialCenter )); // map._initialCenter -> this.map._initialCenter
    },
```

**Comment by xEviL** [comment]
Yes, you're right, thank you for the report. Could you make a fix and corresponding pull request?

**Comment by ZuchaoWang** [comment]
yes, PR submitted!

**Comment by xEviL** [comment]
@ZuchaoWang, thanks! This issue is fixed by 4edb6a241b53620bff83c983e5b13d775dd5ca06
